bundles/org.openhab.binding.mideaac/README.md

```diff
@@ -0,0 +1,76 @@
+## Midea AC Binding
+
+This binding integrates Air Conditioners that use the Midea protocol. Midea is an OEM for many brands.
+
+An AC device is likely supported if it uses one of the following Android apps or it's iOS equivalent.
+ 
+| Application						 		 |				Comment									 |
+|:-------------------------------------------|:------------------------------------------------------|
+|Midea Air (com.midea.aircondition.obm)		 | Full Support of key and token updates				 |
+|NetHome Plus (com.midea.aircondition)		 | Full Support of key and token updates				 |
+|SmartHome/MSmartHome (com.midea.ai.overseas)| Full Support of key and token updates				 |
+
+Note: The Air Conditioner must already be set-up on the WiFi network and must have a fixed IP Address 
+	with one of the three apps listed above for full discovery and key and token updates.
+
+## Supported Things
+
+This binding supports one Thing type `ac`.
+
+## Discovery
+
+Once the Air Conditioner is on the network (WiFi active) it could be discovered automatically.
+An IP broadcast message is sent and every responding unit gets added to the Inbox.
+
+## Binding Configuration
+
+No binding configuration is required.
+
+## Thing Configuration
+
+| Parameter            		| Required ? 		| Comment
+|:--------------------------|:------------------|:----------------------------------------------------------------------------------|
+| IP Address 				| Required 			| IP Address of the device.          												|                                       
+| IP Port         			| Required  		| IP port of the device (for V2: 6444).												|
+| Device ID       			| Required  		| ID of the device. Leave 0 to do ID discovery (length 6 bytes). 					|                                   
+| Cloud Provider 			| Required for V.3	| Cloud Provider name for email and password										|
+| Cloud Provider Email  	| Optional 			| Email for cloud account chosen in Cloud Provider.									|
+| Cloud Provider Password 	| Optional 			| Password for cloud account chosen in Cloud Provider.								|
+| Token  					| Required for V.3 	| Secret Token (length 128 HEX)														|
+| Key						| Required for V.3 	| Secret Key (length 64 HEX)       													|
+| Polling time 				| Required  		| Polling time in seconds. Minimum time is 30 seconds, default is 60 seconds.		|
+| Timeout Socket			| Required 			| Connecting timeout. Minimum time is 2 second, maximum 10 seconds (4 secs default).|
+| Prompt tone 				| Optional 			| "Ding" tone when command is received and executed.								|
+
+## Channels
+
+Following channels are available:
+
+| Channel               | Type                 | Description                                                                                                    | read only |
```

Comment:
The Channel column should contain the `lower-case-hyphen` id, reflecting the thing-type.xml / structure.

Reply:
done